components/tabs/src/sync/bridge.rs

```diff
@@ -279,7 +278,7 @@ mod tests {
     use std::collections::HashMap;
     use sync15::{ClientData, DeviceType, RemoteClient};
 
-    const TTL_1_YEAR: u32 = 31_622_400;
+    const TABS_CLIENT_TTL: u32 = 15_552_000; // same as in storage.rs
```

Comment:
The one in storage.rs is `pub` so just `use` it here? (Not a big deal as it's just tests, but might as well anyway!)